node-services/common/src/common/server-side/jwtTokenUtils.ts

```diff
@@ -40,6 +39,17 @@ export default class JwtTokenUtils {
         return false;
     }
 
+    public getUser(token: string | null): string | undefined {
+        let user;
+        if (token != null && token?.length > 0) {
+            const jwt = this.decodeToken(token);
+            if (jwt != null) {
+                user = jwt.sub;
```

Comment:
The default value for the user login is "sub" but it is possible in opfab to use another value define in common.yml  with the field "security.jwt.login-claim".
You need to take that into account